readthedocs/projects/static-src/projects/js/import.js

```diff
@@ -0,0 +1,252 @@
+var ko = require('knockout'),
+    $ = require('jquery'),
+    tasks = require('../../../../core/static-src/core/js/tasks'),
```

Comment:
Agreed, I hate this pattern as well. It's silly that browserify doesn't include the path it's running from by default, giving you addressable modules of `readthedocs/core/js/whatever.js`. I've patched the browserify instance to handle this and it's much prettier.